PublicMapping/districtbuilder
# Organization project templates

## Overview

Brief description of what this PR does, and why it is needed.

### Checklist


### Demo

New organization template section:
![localhost_3003_o_mikes-maps_ (4)](https://user-images.githubusercontent.com/4432106/108408003-84dc8180-71f2-11eb-9a07-8bd015d78a87.png)

Maps created from a template have a dropdown instead of a ✏:
![localhost_3003_o_mikes-maps_ (5)](https://user-images.githubusercontent.com/4432106/108408011-87d77200-71f2-11eb-986b-3ce4c02465b4.png)

About this template modal:
![localhost_3003_o_mikes-maps_ (6)](https://user-images.githubusercontent.com/4432106/108408019-89a13580-71f2-11eb-90ca-548cc8b6a5e5.png)


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

- `scripts/migration`
- Add project template(s) to your DB, either using SQL or w/ a YAML configuration and the `update-organization` command.
- Create a new map from an organization template on the Organization home screen (`/o/<organization.slug>`) - it should redirect you to the project screen 
- Maps created from templates should show a dropdown menu instead of a ✏ button next to the project name, allowing editing the project name (as normal) or opening a modal to show template details

Closes #227
